### ISSUE TITLE
No way to access color toggle via keyboard navigation - icons.brand.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa801#9736]
# Motivation

Manual testing has revealed that the color toggle is not keyboard accessible on icons.brand.uiowa.edu.

# Additional Info

<img width="1706" height="344" alt="Image" src="https://github.com/user-attachments/assets/b57b9605-354d-4cac-a9f1-692e2ca9cc1f" />

A user should be able to navigate to this element and toggle on/off via the keyboard.
